page-manager.js

```diff
@@ -252,6 +257,8 @@ define([
                     Logger.warn('PageManager: ControllerInfo format is not valid ->', controllerInfo);
                     return;
                 }
+                /** Sets the needAuthentication value for each route as true(default), can be override it in the routes definition */
+                controllerInfo.needAuthentication = controllerInfo.needAuthentication !== false ;
```

Comment:
this flag is set here, but isn't used later.  how about using (in the app custom code, I suppose) a flag called `skipAuthentication` that will be default to `false` hence making it not necessary here?